README.md

```diff
@@ -143,6 +143,26 @@ Modelines for JS files:
 
 One way to profile j2me.js is to use the JS profiler available in Firefox Dev Tools. This will tell us how well the JVM is working and how well natives work. This type of profiling will not measure time that is taken waiting for async callbacks to be called (for example, when using the native JS filesystem API).
 
+## Benchmarks
+
+### Startup Benchmark
+
+The startup benchmark measures from when the benchmark.js file loads to the call of `DisplayDevice.gainedForeground0`. Included in a benchmark build are helpers to build baseline scores so that subsequent runs of the benchmark can be compared. A t-test is used in the comparison to see if the changes were significant.
+
+To use:
+
+*It is recommended that a dedicated Firefox profile is used with the about:config preference of `security.turn_off_all_security_so_that_viruses_can_take_over_this_computer` set to true so garbage collection and cycle collection can be run in between test rounds*
+
+1. Checkout the version you want to be the baseline(usually mozilla/master).
+1. Build a benchmark build `RELEASE=1 BENCHMARK=1 make`
+1. Open the midlet you want to test and click `Build Benchmark Baseline`
+1. When finished the message `FINISHED BUILDING BASELINE` will show up in the console.
```

Comment:
So, you could tell readers to add "&logLevel=log" to the URL parameters (or set the log level in some other way). But even better would be to remove "RELEASE=1" from this set of instructions, since that flag is only peripherally related to benchmarking.